pages.pt_BR/common/gpg.md

```diff
@@ -0,0 +1,32 @@
+# gpg
+
+> GNU Privacy Guard.
+> Mais informações: <https://gnupg.org>.
+
+- Assinar doc.txt sem criptografia (cria um arquivo de saída doc.txt.asc):
+
+`gpg --clearsign {{doc.txt}}`
+
+- Criptografa doc.txt para alice@example.com (cria um arquivo de saída doc.txt.gpg):
+
+`gpg --encrypt --recipient {{alice@example.com}} {{doc.txt}}`
+
+- Criptografa doc.txt com uma senha simétrica somente (cria um arquivo de sadída doc.txt.gpg):
+
+`gpg --symmetric {{doc.txt}}`
+
+- Descriptografa doc.txt.gpg (envia saída para `stdout`):
+
+`gpg --decrypt {{doc.txt.gpg}}`
+
+- Importa uma chave pública:
+
+`gpg --import {{public.gpg}}`
+
+- Exporta  a chave púbica da alice@example.com (envia saída para `stdout`):
```

Comment:
```suggestion
- Exporta a chave púbica da alice@example.com (envia saída para `stdout`):
```

Reply:
I changed it. There is a typo also in "púbica". It should be "pública".